.mk/k8s.mk

```diff
@@ -54,6 +53,8 @@ include .mk/null.mk
 
 include .mk/docker.mk
 
+include .mk/gke.mk
+
```

Comment:
Why we need to move it after docker?

Reply:
To be able do gke-push, not used right now on CI, to reuse all defined containers from make file.